website/docs/r/iot_security_solution.html.markdown

```diff
@@ -0,0 +1,128 @@
+---
+subcategory: "Security Center"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_iot_security_solution"
+description: |-
+  Manages an iot security solution.
+---
+
+# azurerm_iot_security_solution
+
+Manages an iot security solution.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_iothub" "example" {
+  name                = "example-IoTHub"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+
+  sku {
+    name     = "S1"
+    capacity = "1"
+  }
+}
+
+resource "azurerm_iot_security_solution" "example" {
+  name                = "example-Iot-Security-Solution"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+  display_name        = "Iot Security Solution"
+  iothub_ids          = [azurerm_iothub.example.id]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the Iot Security Solution. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) Specifies the name of the resource group in which to create the Iot Security Solution. Changing this forces a new resource to be created.
+
+* `location` - (Required) Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.
+
+* `display_name` - (Required) Specifies the Display Name for this Iot Security Solution.
+
+* `iothub_ids` - (Required) Specifies the IoT Hub resource IDs to which this Iot Security Solution is applied.
+
+* `enabled` - (Optional) Is the Iot Security Solution enabled? Defaults to `true`.
+
+* `events_to_export` - (Optional) A list of data which is to exported to analytic workspace. Valid values include `RawEvents`.
+
+* `log_analytics_workspace_id` - (Optional) Specifies the Log Analytics Workspace ID to which the security data will be sent.
+
+* `log_unmasked_ips_enabled` - (Optional) Should ip addressed be unmasked in the log? Defaults to `false`.
+
+* `recommendations_enabled` - (Optional) A `recommendations_enabled` block as defined below.
+
+* `query_for_resources` - (Optional) Azure Resource Graph query which represents the security solution's monitored resources.
+
+* `query_subscription_ids` - (Optional) A list of subscription Ids on which the user defined resources query should be executed.
+
+* `tags` - (Optional) A mapping of tags to assign to the resource.
+
+---
+
+A `recommendations_enabled` block supports the following:
+
+* `events_to_exportacr_authentication` - (Optional) Could Service Principal Authentication be used with ACR repository? Defaults to `true`.
```

Comment:
We need to fix thes properties:
```suggestion
* `acr_authentication` - (Optional) Is Principal Authentication enabled for the ACR repository? Defaults to `true`.
```

and so forth

Reply:
oh, sorry for all the typo